languages/opml-importer.pot

```diff
@@ -23,7 +23,7 @@ msgid "OPML Importer"
 msgstr ""
 
 #. Plugin URI of the plugin
-msgid "http://wordpress.org/extend/plugins/opml-importer/"
+msgid "https://wordpress.org/extend/plugins/opml-importer/"
```

Comment:
IMHO this pot file should just be deleted entirely, it's no longer needed due to language packs.

Reply:
There appear to be [36 language packs](https://translate.wordpress.org/projects/wp-plugins/opml-importer/language-packs/) for this plugin.

The process to remove the [POT file reference](https://github.com/WordPress/opml-importer/blob/cc9a3ef77c81e071f4d2028dadf0f1e85d926373/opml-importer.php#L249) and file itself could be modeled after this: https://github.com/WordPress/wordpress-importer/pull/9.

Reply:
Language packs are not dependant upon a `.pot` file existing;

Changing the `load_plugin_textdomain()` call as done in https://github.com/WordPress/wordpress-importer/pull/9 just removes the ability to ship translations in the plugin and make it 100% reliant upon language packs; Right now it's in the middle of technically supporting shipped language files, while also supporting language packs.

Reply:
I will admit to being a bit presumptive here, using `wordpress-importer` as the baseline/standard (at least, as of that decision from 2017 😅), where shipped file support was removed "because importers are using language packs."

In reviewing other importers, with the exception of https://github.com/WordPress/blogger-importer, they retain local language files (counter to the above assertion).

Going forward, should importers prefer the pattern used in https://github.com/WordPress/blogger-importer, where there is no `.pot`, but they retain the check for shipped/local files in `load_plugin_textdomain()`?

(Similar updates to this PR were applied in other importers, so clarity here could help address the question of retaining local lang file support, and whether importer `.pot`s should be maintained going forward.)